starlite/handlers/base.py

```diff
@@ -234,6 +240,20 @@ def resolve_opts(self) -> None:
 
         self.opt = opt
 
+    def resolve_signature_namespace(self) -> dict[str, Any]:
+        """Build the route handler signature namespace dictionary by going from top to bottom.
+
+        Names from layers are merged, if multiple layers define the same key, the value from the closest layer to the
+        response handler takes precedence.
```

Comment:
```suggestion
        When merging names from multiple layers, if the same key is defined by multiple layers, the value from the
        layer closest to the response handler will takes precedence.
```

Reply:
Just a suggestion, if it makes sense. It hurt my head reading the original but I am probably dumb 😛

Reply:
Will absolutely update this. I copied the docstring from the `resolve_opts()` method on the same class and modified it a bit, so I'll change that one too.